README.md

```diff
@@ -429,7 +429,7 @@ For questions related to using the library, please visit a support community
 instead of filing an issue on GitHub.
 
 - [Spectrum][spectrum]
-- [Reactiflux on Discord][reactiflux]
+- [Discord][discord]
```

Comment:
Let's ditch the spectrum link too

Reply:
Done, what about the badge though?

Reply:
Let's get rid of all spectrum badges. Is there a discord badge we can use? I've seen some that show how many are online

Reply:
What do you prefer?

[![Discord](https://img.shields.io/discord/723559267868737556?style=flat-square)](https://discord.gg/c6JN9fM)
[![Discord](https://img.shields.io/discord/723559267868737556.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/c6JN9fM)

You can also customize further by clicking on the Discord badge here https://shields.io/category/chat

Reply:
Let's do the second one. Looks cooler.